timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServerLauncher.java

```diff
@@ -73,18 +68,13 @@ public void initialize(Bootstrap<CombinedTimeLockServerConfiguration> bootstrap)
         MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate(
                 "AtlasDbTest" + UUID.randomUUID().toString());
         bootstrap.setMetricRegistry(metricRegistry);
-        bootstrap.setObjectMapper(ObjectMappers.newServerObjectMapper());
-
-        bootstrap.getObjectMapper().registerSubtypes(NonBlockingFileAppenderFactory.class);
-        bootstrap.getObjectMapper().setSubtypeResolver(new DiscoverableSubtypeResolver());
-        bootstrap.getObjectMapper().registerModule(new Jdk8Module());
+        bootstrap.setObjectMapper(ObjectMappers.json());
```

Comment:
We are losing following calls. Is it something we want?
- We don't make a call to _withDefaultModules_
- `.enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)`

Reply:
Ah good catch, maybe there's not as much code to share as I though. I've pushed a fix.